### ISSUE TITLE
Update to KDE Plasma 5.25 for better desktop mode touch support and other new features

### ISSUE THREAD

**Issue by JLP** [ValveSoftware/SteamOS#797]
#### Your system information

* Steam client version: 1654818235 (build )
* SteamOS version: 3.2 (build 20220526.100)
* Opted into Steam client beta?: Yes 
* Opted into SteamOS beta?: Yes 
* Have you checked for updates in Settings > System?: Yes

#### Please describe your issue in as much detail as possible:
The new version of [KDE Plasma 5.25](https://kde.org/announcements/plasma/5/5.25.0/) is available and it brings quite a few touch mode improvements (in addition to other cool new features). Also there are a lot of new features and improvements in [version 5.24](https://kde.org/announcements/plasma/5/5.24.0/) which is also not available yet on Steam Deck. Please update to the latest version in the base Steam OS image so we gett all these improvements. It would be great to also get updated KDE Frameworks.

#### Steps for reproducing this issue:
1. Switch to desktop mode
2. Open application System → Info Center
3. Check KDE Plasma Version, it should report 5.25.0 or greater

**Comment by sevmonster** [comment]
KDE Connect pretty please.

**Comment by JLP** [comment]
> KDE Connect pretty please.

There is a separate issue open for KDE Connect integration: #749

**Comment by Atem18** [comment]
5.26 is out, maybe it's time for an upgrade ?

**Comment by Knochi** [comment]
I need this please! :-)
https://github.com/gerritdevriese/kzones/issues/17

**Comment by Atem18** [comment]
https://store.steampowered.com/news/app/1675200/view/3383920059429575957

Tl;dr: 5.26 is coming as well as kdeconnect

**Comment by JLP** [comment]
Awesome news. Just installed this and yeah so nice to have an up to date KDE Plasma desktop and KDE apps.

BTW I think this new setting should be Enabled by default: System Settings → Workspace Behavior → General Behavior → Touch Mode → Always Enabled. Or even better enabled in handheld mode and disabled when docked (when mouse/keyboard detected)
